src/pages/500.astro

```diff
@@ -11,7 +11,7 @@ import { Heading, Link, Paragraph } from '@amsterdam/design-system-react'
 <Layout title="Er is iets mis gegaan">
   <Heading level={1} className="ams-mb-l">Er is iets mis gegaan</Heading>
   <Paragraph className="ams-mb-m" size="large">De pagina die u probeert te bezoeken heeft een storing.</Paragraph>
-  <Paragraph className="ams-mb--md">
+  <Paragraph className="ams-mb--m">
```

Comment:
In the latest version of the Design System, the double dashes aren't used anymore. Also, the margin bottom of a Paragraph before a Heading is `ams-mb-l`. This goes for all your changes.

```suggestion
  <Paragraph className="ams-mb-l">
```